src/auth/screens/language-settings.js

```diff
@@ -99,6 +99,11 @@ export default [
     emojiCode: ':flag-tw:',
     name: '正體中文',
   },
+  {
+    code: 'ca',
```

Comment:
Alphabetically must be..

Reply:
Oh, I saw the sorting function after the array, so I though the order doesn't matter :)
But I'm going to edit this now, thanks for the feedback.

Reply:
@lex111

Reply:
Ooops :)

then it doesn't matter, I just thought that the code should be consistent...

Reply:
Hmm, I just got all the latest changes from the master branch and tried to build the application, everything was OK
MacOS High Sierra